lib/export.js

```diff
@@ -48,11 +48,11 @@ class ES6Exporter {
       } else {
         output.push(`import ${def.basedOn[0].name} from '${relativeImportPath(def.identifier, def.basedOn[0])}';\n\n`)
```

Comment:
Since you're sanitizing the filenames and classnames, I think you also need to sanitize `def.basedOn[0].name` in the import statement here.